tests/test_features.py

```diff
@@ -958,10 +958,21 @@ def test_mfcc_to_mel(n_mfcc, n_mels, dct_type):
                                             dct_type=dct_type,
                                             lifter=10**-3)
 
-    # check if mfcc_to_mel it works correctly with lifter
-    librosa.feature.inverse.mfcc_to_mel(mfcc, n_mels=n_mels,
-                                        dct_type=dct_type,
-                                        lifter=1)
+    # check if mfcc_to_mel works correctly with lifter
+    ones = np.ones(mfcc.shape)
+    negate_lifter_vec = 1 + (lifter / 2) * np.sin(np.pi * np.arange(1, 1 +
```

Comment:
this nonsyntactic line break is difficult to parse for the human reader. it breaks at the worst possible place in the equation, at the sixth level of nesting.

Reply:
the line breaks are automatically done using [autopep8](https://github.com/hhatto/autopep8). Although, this is practical but I admit that it can result in some weird line breaks. [yapf](https://github.com/jpadilla/yapf-online) seems to provide better results. Shortening the variable name and using yapf results in: 
```
    lifter_vec = 1 + (lifter / 2) * np.sin(np.pi * np.arange(
        1, 1 + n_mfcc, dtype=mfcc.dtype) / lifter)[:, np.newaxis]
```
If you think this variant is better, I will edit the code accordingly. Otherwise, I suggest a no line break at all in this position.

Reply:
you can also assign a temporary variable for `np.arange(1, 1 + n_mfcc, dtype=mfcc.dtype)`